packages/arb-token-bridge-ui/src/util/NumberUtils.ts

```diff
@@ -54,6 +54,14 @@ export const formatAmount = <T extends number | BigNumber>(
 
   // Small number, show 4 or 5 decimals based on token name length
   if (value < 1) {
+    const maximumFractionDigits = isShortSymbol
+      ? MaximumFractionDigits.Long
+      : MaximumFractionDigits.Standard
+    const maxValue = Number(`1e${-maximumFractionDigits}`)
```

Comment:
1. I think `minValue` is more accurate here?
2. NIT: `Math.pow(10, -maximumFractionDigits)` could be a little bit more clear here.